### DISCUSSION TITLE
Access denied for user 'XXXXXXX'@'localhost' (using password: YES)

### DISCUSSION THREAD

**Discussion by ps97-ketangohil** [Nikeev/sesdashboard#46] [Q&A]
A message "Access denied for user 'XXXXXXX'@'localhost' (using password: YES)" message comes when I am executing **./bin/console doctrine:migrations:migrate -n**

Please advise a fix. 

Error snapshot is attached.

![image](https://user-images.githubusercontent.com/22788034/176632211-883ddecf-15f2-4426-b8aa-0c2f5674802c.png)

**Comment by Nikeev** [comment]
Hello! It seems that you provided incorrect MySQL credentials in .env.local
Please check that MYSQL_ variables are configured correctly and you db user has access to db.
Do you run SesDashboard in Docker or without?

**Reply by ps97-ketangohil** [reply]
Hi Nikeev,

Thank you for your response.

The credentials are correct in the .env.local file. The credentials are tested multiple times.

It is not a docker. It is a linux VM (Ubuntu 22.04 LTS) created on Azure.

It has something to do with the type of password I think.

One more strange thing noticed. As per document we should copy .env to .env.local which I did but actually it picks the connection from the .env file only.

Regards,
Ketan
